hydra-cardano-api/src/Hydra/Cardano/Api/TxOutValue.hs

```diff
@@ -7,8 +7,9 @@ import Hydra.Cardano.Api.MaryEraOnwards (IsMaryEraOnwards (..))
 -- | Inject some 'Value' into a 'TxOutValue'
 mkTxOutValue ::
   forall era.
+  IsShelleyBasedEra era =>
   IsMaryEraOnwards era =>
```

Comment:
Do we need to keep both constraints here?

Reply:
Yes. Neither implies the other